devops/iam-did-auth-proxy-dev/values.yaml

```diff
@@ -127,7 +127,7 @@ did-auth-proxy-helm:
     serverBlock: |-
       map $http_origin $allow_origin {
         default "";
-        "~^(http://localhost:4200|http://localhost:4201|https://gp4btc-miner-dev.energyweb.org|https://gp4btc-explorer-dev.energyweb.org)$" "$http_origin";
+        "~^(https://gp4btc-miner.energyweb.org,https://explorer.gp4btc.org,https://miner.gp4btc.org)$" "$http_origin";
```

Comment:
These values are the prod values. I don't think they should be used for `devops/iam-did-auth-proxy-dev/values.yaml`, right?